File: Python/String-1/without_end.py
"""

Given a string, return a version without the first and last char, so "Hello" yields "ell". The string length will be at least 2.


without_end('Hello') → 'ell'
without_end('java') → 'av'
without_end('coding') → 'odin'

@author unobatbayar

"""

def without_end(str):
  first = str[1:len(str)-1]
  return first

File: CodingBat/Java/String-1/withoutEnd.java
/**
 * Given a string, return a version without the first and last char, so "Hello" yields "ell". The string length will be at least 2.

withoutEnd("Hello") → "ell"
withoutEnd("java") → "av"
withoutEnd("coding") → "odin"
@author unobatbayar
 */
public String withoutEnd(String str) {
    if(str.length()<2) return str;
    return str.substring(1, str.length()-1);
  }
  